CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 - feat(github): Retry on 404s (#177)
 - ref(aws-lambda): Catch potential exceptions when publishing AWS Lambda layers to new regions (#178)
+- ref(aws-lambda): Add runtime names on commit message (#181)
```

Comment:
I'd argue this is a `feat` or a `fix` not a ref. See the definitions here: https://develop.sentry.dev/code-review/#type